docs/release/release-tasks.md

```diff
@@ -290,6 +290,8 @@ for provider authors on how to adopt to the new Cluster API version.
 
 1. Update supported versions in versions.md.
    <br>Prior art: [Update supported versions for v1.3](https://github.com/kubernetes-sigs/cluster-api/pull/6850)
+2. Update Cluster API support and guarantees in CONTRIBUTING.md.
+   <br>Prior art: [Support and guarantees for 1.4](https://github.com/kubernetes-sigs/cluster-api/pull/8308)
```

Comment:
Should we drop step 6 of `Cut a release`? (I think it's the same)

Reply:
Also not sure if it fits better here as this task is done in week 1 and not after the release which was the original intention (but no strong opinion, up to you)

Reply:
Another suggestion would be to reword step 6 in `Cut a release` to also explicitly mention "update support and guarantees" and also add a note that this step should only be done for minor releases.
(similar to how we have a note that the homebrew step (step 5) should only be done for latest stable release and not for other patch releases.)